commands/sendReminderCheckinEmails.js

```diff
@@ -29,7 +29,7 @@ async function run() {
         && offset < 72 * 3600 * 1000
         && !lu.deleted) {
         const hasLocalPhoneNumber = false;
-        const inCountry = await user.isInCountry(lu.list.metadata.country.pcode);
+        const inCountry = false;
```

Comment:
I think that could be removed as well as the other references of `inCountry` in https://github.com/UN-OCHA/hid_api/blob/e29f471b98547758573d0e9fb244fad34d9f9101/emails/reminder_checkin/html.ejs#L9 and https://github.com/UN-OCHA/hid_api/blob/e29f471b98547758573d0e9fb244fad34d9f9101/emails/reminder_checkin/fr/html.ejs#L9

Reply:
Whoops, perhaps I could have set the var to `true` to ensure the email never displays that part of the message.

At any rate, we're dropping all of the check-in code "soon" (I might get to it today) so by the time we deploy these changes, it will get expunged.